exchange/utils.go

```diff
@@ -984,21 +984,25 @@ func getMediaTypeForBid(bid openrtb2.Bid) (openrtb_ext.BidType, error) {
 }
 
 func getPrebidMediaTypeForBid(bid openrtb2.Bid) (openrtb_ext.BidType, error) {
+	var err error
+	var bidType openrtb_ext.BidType
```

Comment:
I noticed that these are declared out here, I'm interested as to why you made this change. I'm wondering if the error handling on lines 1000-1003 could be handled when the `err` is captured on line 994. What do you think about this?

Reply:
Yes, this change is discussed here: https://github.com/prebid/prebid-server/pull/2721#discussion_r1196705940
We need to return actual error and don't overwrite it with custom message. 
These variables are needed for correct data handling. Please note test `TestGetPrebidMediaTypeForBid` and differences in error messages.